### PR TITLE
PIM-4091: category tree css

### PR DESCRIPTION
**Pull request by yulldesign**

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 1]
```diff
@@ -6,25 +6,71 @@
...
+      border: none;
+      .jstree-icon{
+        background:url("jstree/folder_selected_opened.png") 0 0 no-repeat;
+      }
+    }
   }
-  li.jstree-closed > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_standard_closed.png") 0 0 no-repeat;
+  
+  li.jstree-closed{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 2]
```diff
@@ -6,25 +6,71 @@
...
-
-  li.jstree-open > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_standard_opened.png") 0 0 no-repeat;
+  
+  li.jstree-open{
+    > a:not(.jstree-loading) .jstree-icon{
+      background:url("jstree/folder_standard_opened.png") 0 0 no-repeat;
+    }
+    
+    > a.jstree-clicked{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 3]
```diff
@@ -6,25 +6,71 @@
...
+  
+  .jstree-clicked{
+    padding: 1px 2px;
   }
-
-  li.jstree-open > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_standard_opened.png") 0 0 no-repeat;
+  
+  li.jstree-open{
+    > a:not(.jstree-loading) .jstree-icon{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 4]
```diff
@@ -6,25 +6,71 @@
...
+      background:url("jstree/folder_standard_opened.png") 0 0 no-repeat;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `:`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 5]
```diff
@@ -6,25 +6,71 @@
...
-    border: 1px solid darken(@color-purple_hover, 5%);
+  
+  .jstree-clicked{
+    padding: 1px 2px;
   }
-
-  li.jstree-open > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_standard_opened.png") 0 0 no-repeat;
+  
+  li.jstree-open{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 6]
```diff
@@ -6,25 +6,71 @@
...
+    background: none;
+    border: none;
+    color: @color-new-purple-medium;
+    padding: 1px 2px;
   }
-  .jstree-clicked {
-    background: darken(@color-purple_hover, 5%);
-    border: 1px solid darken(@color-purple_hover, 5%);
+  
+  .jstree-clicked{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 7]
```diff
@@ -6,25 +6,71 @@
...
+  li.jstree-open{
+    > a:not(.jstree-loading) .jstree-icon{
+      background:url("jstree/folder_standard_opened.png") 0 0 no-repeat;
+    }
+    
+    > a.jstree-clicked{
+      color: @color-new-purple-medium;
+      background: none;
+      border: none;
+      .jstree-icon{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 8]
```diff
@@ -6,25 +6,71 @@
...
+        background:url("jstree/folder_selected_opened.png") 0 0 no-repeat;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `:`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 9]
```diff
@@ -6,25 +6,71 @@
...
+      }
+    }
   }
-  li.jstree-closed > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_standard_closed.png") 0 0 no-repeat;
+  
+  li.jstree-closed{
+    > a:not(.jstree-loading) .jstree-icon {
+      background:url("jstree/folder_standard_closed.png") 0 0 no-repeat;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `:`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 10]
```diff
@@ -6,25 +6,71 @@
...
+    }
+    > a.jstree-clicked{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 11]
```diff
@@ -6,25 +6,71 @@
...
+      color: @color-new-purple-medium;
+      background: none;
+      border: none;
+      .jstree-icon{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 12]
```diff
@@ -6,25 +6,71 @@
...
+  li.jstree-leaf{
+    > a:not(.jstree-loading) .jstree-icon {
+      background:url("jstree/folder_leaf.png") 0 0 no-repeat;
+    }
+    > a.jstree-clicked{
+      color: @color-new-purple-medium;
+      background: none;
+      border: none;
+      .jstree-icon{
+        background:url("jstree/folder_selected_leaf.png") 0 0 no-repeat;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `:`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 13]
```diff
@@ -6,25 +6,71 @@
...
+      }
+    }
   }
-  li.jstree-unclassified > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_unclassified.png") 0 0 no-repeat;
+  
+  li.jstree-unclassified{
+    > a:not(.jstree-loading) .jstree-icon {
+      background:url("jstree/folder_unclassified.png") 0 0 no-repeat;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 14]
```diff
@@ -6,25 +6,71 @@
...
+      border: none;
+      .jstree-icon{
+        background:url("jstree/folder_selected_leaf.png") 0 0 no-repeat;
+      }
+    }
   }
-  li.jstree-unclassified > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_unclassified.png") 0 0 no-repeat;
+  
+  li.jstree-unclassified{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 15]
```diff
@@ -6,25 +6,71 @@
...
+    > a:not(.jstree-loading) .jstree-icon {
+      background:url("jstree/folder_unclassified.png") 0 0 no-repeat;
+    }
+    > a.jstree-clicked{
+      color: @color-new-purple-medium;
+      background: none;
+      border: none;
+      .jstree-icon{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 16]
```diff
@@ -6,25 +6,71 @@
...
+        background:url("jstree/folder_selected_closed.png") 0 0 no-repeat;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `:`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 17]
```diff
@@ -56,3 +102,7 @@
     padding-left: 9px;
   }
 }
+
+.jstree li{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add spaces before `{`

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 18]
```diff
@@ -56,3 +102,7 @@
...
+  line-height: 28px;
+}
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a line at the end of the file (you really should configure your editor to do that automatically or switch to a real editor :))

Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: missing blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less (hunk 7 above)
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a blank line before

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less (hunk 10 above)
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a blank line before

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less (hunk 11 above)
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a blank line before

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 19]
```diff
@@ -6,25 +6,71 @@
...
+    }
   }
-  li.jstree-leaf > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_leaf.png") 0 0 no-repeat;
+  
+  li.jstree-leaf{
+    > a:not(.jstree-loading) .jstree-icon {
+      background:url("jstree/folder_leaf.png") 0 0 no-repeat;
+    }
+    > a.jstree-clicked{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a blank line before

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 20]
```diff
@@ -6,25 +6,71 @@
...
+    }
   }
-  li.jstree-unclassified > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_unclassified.png") 0 0 no-repeat;
+  
+  li.jstree-unclassified{
+    > a:not(.jstree-loading) .jstree-icon {
+      background:url("jstree/folder_unclassified.png") 0 0 no-repeat;
+    }
+    > a.jstree-clicked{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a blank line before

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less (hunk 15 above)
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a blank line before

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 21]
```diff
@@ -6,25 +6,78 @@
...
-    border: 1px solid darken(@color-purple_hover, 5%);
+    padding: 1px 2px;
   }
-
-  li.jstree-open > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_standard_opened.png") 0 0 no-repeat;
+  
+  li.jstree-open {
+    > a:not(.jstree-loading) .jstree-icon {
+      background:url("jstree/folder_standard_opened.png") 0 0 no-repeat;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a space AFTER `:` ^^

File: src/Pim/Bundle/UIBundle/Resources/public/css/tree.less [hunk 22]
```diff
@@ -6,25 +6,78 @@
...
+        background: url("jstree/folder_selected_leaf.png") 0 0 no-repeat;
+      }
+    }
   }
-  li.jstree-unclassified > a:not(.jstree-loading) .jstree-icon {
-    background:url("jstree/folder_unclassified.png") 0 0 no-repeat;
+  
+  li.jstree-unclassified {
+    > a:not(.jstree-loading) .jstree-icon {
+      background:url("jstree/folder_unclassified.png") 0 0 no-repeat;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tree.less]: please add a space after `:`

**Comment by sumbobyboys** [comment]
GTM

**Pull request merged**